components/RichText/Quote.tsx

```diff
@@ -25,7 +25,7 @@ export function Quote({ node, children }: PropsWithChildren<Props>) {
                     [styles.alignRight]: node.align === Alignment.RIGHT,
                 })}
             >
-                {children}
+                <p>{children}</p>
```

Comment:
Was there a particular reason for this paragraph tag to be added? I get extra margin on top and bottom from the default browser stylesheet on Chrome (Mac). If this is what you intended, it is better to add these styles explicitly, since we don't want to rely on user agent stylesheets.

Reply:
that was to apply the p/paragraph styles, will change to explicitly use the paragraph class